Challenge IV/indexIV.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+		<link rel="stylesheet" href="styleIV.css">
+	</head>
+	<body>
+		<p>Name
+  			<input type="text" id="name"><br>
+  			Favorite Food
+  			<input type="text" id="favoriteFood"><br>
+  			Favorite Color
+  			<input type="text" id="favoriteColor"><br>
+  			<button id="button">Submit</button>
```

Comment:
It's a good idea to use a unique name for your IDs and Classes (for example, "button" is also used as an HTML element `<button>`, so it's not the best name for an ID.) Maybe an id like "addToTable" would work better.